gtsfm/frontend/correspondence_generator/det_desc_correspondence_generator.py

```diff
@@ -2,65 +2,116 @@
+            gt_cameras: GT cameras, used to evaluate metrics.
+            gt_scene_mesh: GT mesh of the 3D scene, used to evaluate metrics.
+            two_view_estimator: two view estimator, which is used to verify correspondences and estimate pose.
+
+        Returns:
+            List of keypoints, one entry for each input images.
+            Two view output for image_pairs.
         """
-        # Detection and description graph.
-        delayed_keypoints = []
-        delayed_descriptors = []
-        for delayed_image in delayed_images:
-            (delayed_dets, delayed_descs) = self._feature_extractor.create_computation_graph(delayed_image)
-            delayed_keypoints.append(delayed_dets)
-            delayed_descriptors.append(delayed_descs)
-
-        delayed_putative_corr_idxs_dict: Dict[Tuple[int, int], Delayed] = {}
-        for (i1, i2) in image_pair_indices:
-            # Graph for matching to obtain putative correspondences.
-            delayed_putative_corr_idxs_dict[i1, i2] = self._matcher.create_computation_graph(
-                keypoints_i1_graph=delayed_keypoints[i1],
-                keypoints_i2_graph=delayed_keypoints[i2],
-                descriptors_i1_graph=delayed_descriptors[i1],
-                descriptors_i2_graph=delayed_descriptors[i2],
-                im_shape_i1=delayed_image_shapes[i1],
-                im_shape_i2=delayed_image_shapes[i2],
+
+        def apply_det_desc(det_desc: DetectorDescriptorBase, image: Image) -> Tuple[Keypoints, np.ndarray]:
+            return det_desc.detect_and_describe(image)
+
+        def apply_matcher_and_two_view_estimator(
+            feature_matcher: MatcherBase,
+            two_view_estimator: TwoViewEstimator,
+            features_i1: Tuple[Keypoints, np.ndarray],
+            features_i2: Tuple[Keypoints, np.ndarray],
+            im_shape_i1: Tuple[int, int, int],
+            im_shape_i2: Tuple[int, int, int],
+            camera_intrinsics_i1: CALIBRATION_TYPE,
+            camera_intrinsics_i2: CALIBRATION_TYPE,
+            i2Ti1_prior: Optional[PosePrior],
+            gt_camera_i1: Optional[CAMERA_TYPE],
+            gt_camera_i2: Optional[CAMERA_TYPE],
+            gt_scene_mesh: Optional[Any] = None,
+        ) -> TWO_VIEW_OUTPUT:
+            putative_corr_idxs = feature_matcher.match(
+                features_i1[0], features_i2[0], features_i1[1], features_i2[1], im_shape_i1, im_shape_i2
             )
 
-        return delayed_keypoints, delayed_putative_corr_idxs_dict
+            return two_view_estimator.run_2view(
+                features_i1[0],
+                features_i2[0],
+                putative_corr_idxs,
+                camera_intrinsics_i1,
+                camera_intrinsics_i2,
+                i2Ti1_prior,
+                gt_camera_i1,
+                gt_camera_i2,
+                gt_scene_mesh,
+            )
+
+        det_desc_future = client.scatter(self._detector_descriptor, broadcast=False)
```

Comment:
what does `broadcast=False` do?

Reply:
From dask's docstring:
> Additionally the scatter function supports a broadcast= keyword argument to enforce that all the data is sent to all workers rather than round-robined. If new workers arrive they will not automatically receive this data.

I am using False to avoid the scenario with new workers.